lib/cachext/features/circuit_breaker.rb

```diff
@@ -0,0 +1,27 @@
+module Cachext
+  module Features
+    module CircuitBreaker
+      attr_reader :breaker
+
+      def initialize config
+        super
+        @breaker = Breaker.new(config)
+      end
+
+      def read key, options
+        circuit = breaker.for(key)
+        if circuit.open?
+          key.read_backup
```

Comment:
Yea. It'll return `nil`, or if the `default` option is set it return/call that (like `default: -> { [] }`)